packages/pluggableWidgets/slider-web/src/utils/getPreviewValues.ts

```diff
@@ -0,0 +1,12 @@
+import { SliderPreviewProps } from "../../typings/SliderProps";
+
+interface GetPreviewValuesParams extends Omit<SliderPreviewProps, "class"> {}
```

Comment:
This doesn't change anything right?

Reply:
Yep, @diego-antonelli , said that this is type generator issue and till we fix it we should manually overwrite "class" prop

Reply:
Yes, this is typing issue, class should be replaces with className in editorPreview file.